electrolyzer/stack.py

```diff
@@ -309,11 +331,16 @@ def update_dynamics(self, H2_mfr_ss, stack_state):
 
         This is really just a filter on the steady state mfr from time step to time step
         """
-        x_k = stack_state
-        x_kp1 = self.DTSS[0] * x_k + self.DTSS[1] * H2_mfr_ss
-        y_kp1 = self.DTSS[2] * x_k + self.DTSS[3] * H2_mfr_ss
-        next_state = x_kp1
-        H2_mfr_actual = y_kp1
+
+        if not self.ignore_dynamics:
```

Comment:
small nitpick, but for clarity consider swapping if/else to avoid double negative